### DISCUSSION TITLE
CPU 100%

### DISCUSSION THREAD

**Discussion by KaKi87** [tsukasa/BdBrowser#36] [Problems and issues]
Hi,

Since today, opening Discord makes CPU usage go to 100% instantly, and disabling the extension is the only way to open Discord without this happening.

Thanks

**Comment by tsukasa** [comment]
Greetings,

I can not confirm the reported behaviour on my end. The high CPU usage you experience might be caused by an installed BetterDiscord plugin rather than the extension itself.

To verify, you can disable all plugins via the extension options and reload the Discord tab.

Please let me know if running without enabled plugins mitigates the issue.

**Reply by KaKi87** [reply]
Sorry I didn't know of this option, it worked indeed.

Now, how can I find out which plugin caused the issue ? I have 50 of these :sweat_smile: 

Thanks

**Reply by tsukasa** [reply]
Greetings,

the only way I can think of is to enable them one by one until the CPU usage is permanently at >= 90% again. I have no knowledge of a more elegant way to pinpoint the offending file more effectively.

There is also the (admittedly unlikely) possibility that two plugins might interfere with each other.
